zest/releaser/utils.py

```diff
@@ -695,9 +695,17 @@ def _execute_command(command, input_value=''):
     stderr_output = e.read()
     # We assume that the output from commands we're running is text.
     if not isinstance(stdout_output, six.text_type):
-        stdout_output = stdout_output.decode(OUTPUT_ENCODING)
+        try:
+            stdout_output = stdout_output.decode(OUTPUT_ENCODING)
+        except UnicodeDecodeError:
+            stdout_output = '%s stdout output is not a valid %s stream.' % (
```

Comment:
> Is it a good thing to silence the error?

To be honest I don't think we should crash on UnicodeDecodeError then my solution might not be the right one, we should talk about it and see how we want to handle it.